### PR TITLE
Added danvgar profile and map location

### PR DESCRIPTION
**Pull request by danvgar**
## Linked Issue
[https://github.com/Virtual-Coffee/virtualcoffee.io/issues/13](https://github.com/Virtual-Coffee/virtualcoffee.io/issues/13)

## Code of Conduct

> By submitting this pull request, you agree to follow our [Code of Conduct](https://virtualcoffee.io/code-of-conduct/)

### CODE REVIEW

File: members/members/danvgar.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import type { MemberObject } from '../types';
+import { profileMasks } from '../flare';
+
+// Change _EXAMPLE to `yourGitHubUserName` and add your info below
+export const _EXAMPLE: MemberObject = {
```
Comment (danieltott) [inline members/members/danvgar.ts:5]: This should be updated to match your username:

```suggestion
export const danvgar: MemberObject = {
```

**Review by danieltott — CHANGES_REQUESTED** [review]
Thanks @danvgar - just one quick change and we should be good to go 👍🏼

File: members/members/danvgar.ts (hunk 1 above)
Comment (danvgar) [reply members/members/danvgar.ts]: Oo, thanks for catching that @danieltott ! Updated in the last commit, let me know if you notice anything else.

**Review by danieltott — APPROVED** [review]

**Pull request merged**